docker-compose.yml

```diff
@@ -33,7 +33,7 @@ services:
       - POSTGRES_USER=postgres
       - POSTGRES_PASSWORD={DB_PASSWORD}
     volumes:
-      - ./data/:/data
+      - ./database/:/database
```

Comment:
This line in docker-compose loads the database folder into the pgdb container so we can access its contents :-)